src/components/Request/FormFields/JOAIdealCandidate.tsx

```diff
@@ -0,0 +1,47 @@
+import { Label, Text, Textarea } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+
+const JOAIdealCandidate = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        htmlFor={name + "Id"}
+        size="small"
+        weight="semibold"
+        className="requestFieldLabel"
+        required
+      >
+        <TextFieldIcon className="requestFieldIcon" />
+        Ideal Candidate
+      </Label>
+      <Controller
+        name="joaIdealCandidate"
+        control={form.control}
+        rules={{
+          required: "Ideal Candidate text is required",
+        }}
+        render={({ field }) => (
+          <Textarea
+            {...field}
+            resize="vertical"
```

Comment:
Don't specify both resize, and a fixed style height, as it makes a resize icon appear, that does nothing but slide up and down the fixed height.

Reply:
Looks like you can leave resize, remove style, and add a rows={10} to have it default to a taller box.